ui/src/variables/actions/index.ts

```diff
@@ -104,7 +105,9 @@ export const createVariable = (variable: Variable) => async (
   } catch (e) {
     console.log(e)
     dispatch(notify(createVariableFailed()))
+    return
   }
+  dispatch(notify(createVariableSuccess(variable.name)))
```

Comment:
_nit_ It might be easier to read the control here if this was right after line 104 (in the "happy path" for the request)